### PR TITLE
add more details to the book README

### LINKED ISSUE #5: docs: add more details to 'Working the Command Line' README
- [x] Add about the book
- [x] Add about the author
- [x] Add about the sponsor
- [x] Include a list of participants
- [x] Include visual aids
- [x] Create a banner
- [x] Keep it light-hearted and fun

### PR DESCRIPTION
**Pull request by bpires**
- [x] Add about the book
- [x] Add about the author
- [x] Add about the sponsor
- [x] Include a list of participants
- [x] Include visual aids
- [x] Create a banner
- [x] Keep it light-hearted and fun

Fixes #5

### CODE REVIEW

File: working-the-command-line/README.md [hunk 1]
```diff
@@ -1,5 +1,81 @@
...
+![i](https://user-images.githubusercontent.com/86871991/189595823-e218bc7a-4d1b-47a0-b714-b1865ef2a360.png)
+
+
+# 📚 _Working the Command Line_, by [Remy Sharp](https://remysharp.com/) (2016)
+
+> BookClub cohort sponsored by [A Book Apart](https://abookapart.com/), _"brief books for people who design, write, and code"_. 
+
+
+
+<br>
```
Comment (schalkneethling) [inline working-the-command-line/README.md:10]: Do we _need_ these `br` tags?

File: working-the-command-line/README.md [hunk 2]
```diff
@@ -1,5 +1,81 @@
...
+* [🌱 The Participants](#-the-participants)
+
+
+<br/>
+
+## 📙 About the book
+
+<div display="inline-block">
+<img align="left" width="125em" src="https://i.pinimg.com/736x/be/19/3f/be193ff8995c39cecbf58c7eeebbc3a9.jpg">
+</div>
```
Comment (schalkneethling) [inline working-the-command-line/README.md:27]: I reckon we can download these images and add them directly to the repository. It feels a little safer. For one thing, we can be sure the image will always render. The other thing is, someone might switch up the image on us :) Also, if we do not _need_ the HTML, I generally prefer to keep the Markdown to just Markdown. It allows us to, for example, add a linter without having to mess too much with formatting rules and exceptions.

File: working-the-command-line/README.md [hunk 3]
```diff
@@ -1,5 +1,81 @@
...
+It offers a gentle approach but spans over many topics, starts with navigating your computer in the terminal, to tricks and shortcuts, to installing software, to string manipulation techniques (which the command line is perfect for), to techniques on "how to shoot yourself in the foot" and finally how to customise your terminal.
+
+<br/>
+
+## 👤 About the author
+
+<div display="inline-block">
+<img align="left" width="100em" src="https://cdn.shopify.com/s/files/1/0051/7692/t/5/assets/author-remy-sharp-1608310970747.jpg">
+</div>  
+
```
Comment (schalkneethling) [inline working-the-command-line/README.md:42]: Same here as my previous comment

File: working-the-command-line/README.md [hunk 4]
```diff
@@ -1,5 +1,81 @@
...
+Remy Sharp has been working commercially on the web since 1999 and blogging for over a decade. He runs his own consultancy, has been writing JavaScript long before it was cool, and founded the web conference ffconf in 2009. 
+
+Remy tends to attract bugs and he enjoys squishing them. He uses the terminal daily.
+
+<br/>
```
Comment (schalkneethling) [inline working-the-command-line/README.md:47]: Ditto :)

File: working-the-command-line/README.md [hunk 5]
```diff
@@ -1,5 +1,81 @@
...
+
+
+## 🌠 About the sponsor
+
+<div display="inline-block">
+<img align="left" width="80em" src="https://media-exp1.licdn.com/dms/image/C560BAQGBqOQVMOSczw/company-logo_200_200/0/1519896899735?e=2147483647&v=beta&t=UcXA2wEcZhr_on_2GhnE4LOLB4jYPCsbexIcRFYjYXU">
+</div>
```
Comment (schalkneethling) [inline working-the-command-line/README.md:54]: Ditto

File: working-the-command-line/README.md [hunk 6]
```diff
@@ -1,5 +1,81 @@
...
+## 🌱 The Participants
+
+
+|   |   |   |  
+|:---:|:---:|:---:|
+|<img width="85em" src="https://cdn.discordapp.com/avatars/748782811481243659/3a820d91a661a32184f027ede66ab52a.png?size=1024">&nbsp;|<img width="85em" src="https://cdn.discordapp.com/avatars/690807885617430558/d425f4cc87577004cd2888701d23d291.png?size=1024">|<img width="85em" src="https://cdn.discordapp.com/avatars/595880225846329345/c300fc8794de8266290aa1858a8d742e.png?size=1024">|
+|Zoe Sullivan (s/t)#0041|rafaelbpires#5725|Shriya Singh#9418|
+|<img width="85em" src="https://cdn.discordapp.com/avatars/199232269406175242/d7464363dc90d7abf8ec47a26e2496c5.png?size=1024">|<img width="85em" src="https://cdn.discordapp.com/avatars/800239929484771349/e94d2422b42da9d6d82a60398bd4a11a.png?size=1024">|
+|Hoodini#7495|MortimerShore#0708|
+
```
Comment (schalkneethling) [inline working-the-command-line/README.md:81]: Again here, perhaps better to add the images directly to the repo.

**Review by schalkneethling — CHANGES_REQUESTED** [review]
Thank you so much for taking the time to put this together @bpires 🎉 A couple of notes and suggestions.

File: working-the-command-line/README.md (hunk 1 above)
Comment (bpires) [reply working-the-command-line/README.md:10]: Fair point!

File: working-the-command-line/README.md (hunk 2 above)
Comment (bpires) [reply working-the-command-line/README.md:27]: Thanks for pointing this out! It feels safer, indeed. And on that note, I think it's best to reduce the number of images to the minimum. I should have given this more thought 😅
In relation to HTML, I couldn't find a way to reduce the image and align it without recurring to the  tag. Please let me know if I'm missing something

**Comment by bpires** [comment]
Thanks for the review @schalkneethling 🍄 I did some changes and remodelled the README based on your suggestions. I also included the latest changes in a new section. Please let me know what you think about it!

**Comment by schalkneethling** [comment]
@bpires Do you have the source image you used for the header? Also, a link to the person’s Etsy store. Thanks!

**Comment by bpires** [comment]
@schalkneethling it's https://www.etsy.com/shop/AshElizabethArt
And the image https://i.etsystatic.com/15228211/r/il/368ea9/2967627473/il_794xN.2967627473_tgil.jpg

**Comment by schalkneethling** [comment]
@bpires what about something like this for the header template:
https://design.penpot.app/#/view/c74f9c80-3778-11ed-a5c2-cf2300df1f4f?page-id=c74f9c81-3778-11ed-a5c2-cf2300df1f4f&section=interactions&index=0&share-id=08802d90-3923-11ed-a5c2-cf2300df1f4f

**Comment by bpires** [comment]
Loved the header @schalkneethling! Just two things:
1. There seems to be an issue with the font weight on the "network" word of the TMN logo
2. You may consider including the author of the book (like "Working the Command Line, by Remy Sharp").

Anyway, LGTM! :)

**Comment by schalkneethling** [comment]
> [Something like this](https://www.canva.com/design/DAFM0MvHEBI/O9BnAMFD0aG0EW-fSv1cNA/edit?utm_content=DAFM0MvHEBI&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton) (Don't know how to use penpot yet, just a draft on Canvas to illustrate).

I really like that! I will update the one on PenPot.

**Comment by schalkneethling** [comment]
@bpires This looks great! We just need to update the header graphic. Thanks!

**Comment by bpires** [comment]
@schalkneethling, I've updated the banner and added links to the individual notes and resources of each chapter. I think we're good to go.

PS: The existent content regarding chapter 4 will be moved to the appropriate markdown file under chapter4 folder.

**Review by schalkneethling — CHANGES_REQUESTED** [review]
Just two more items, almost there 🙌 - Thanks so much, @bpires, I cannot wait for this one to land.

File: working-the-command-line/README.md [hunk 7]
```diff
@@ -1,23 +1,62 @@
-# Working the command line
+![Bookclub banner](./assets/bookclub-banner.png)
```
[the remark this answers is not in the document]
Comment (bpires) [reply working-the-command-line/README.md:1]: Good to know, thanks!

File: working-the-command-line/README.md [hunk 8]
```diff
@@ -1,23 +1,62 @@
-# Working the command line
+![Bookclub banner](assets/bookclub-banner.png)
```
Comment (schalkneethling) [inline working-the-command-line/README.md:1]: @bpires I think these will work once we merge the PR. IF not, we can figure out why it does not then.

**Comment by bpires** [comment]
I've accepted the suggestions and resized the image, leaving all in markdown with no HTML. 🎉 

PS: I'm curious: how do you preview the file output? Because it rendered fine for me here on GitHub with the `./` at the beginning of the file path. I'm asking just to be sure, I guess either way works.

**Comment by schalkneethling** [comment]
> PS: I'm curious: how do you preview the file output? Because it rendered fine for me here on GitHub with the `./` at the beginning of the file path. I'm asking just to be sure, I guess either way works.

I was using "view rich diff", but it seems the "view file" option is the better one.

**Review by schalkneethling — APPROVED** [review]
Thank you for persisting with this one @bpires 🙌

**Pull request merged**